### ISSUE TITLE
Implement support for secure desktops

### ISSUE THREAD

**Issue by leonardder** [leonardder/nvdaRd#3]
There is a long standing NVDA issue about braille on secure screens, see https://github.com/nvaccess/nvda/issues/2315
With the protocol in this add-on, it should be pretty trivial to implement this. As soon as the user session enters a secure desktop, it should start a Named Pipe Server (the basics for named pipe servers are already implemented in addon/lib/namedPipe.py). The NVDA instance on the secure screen should connect to that pipe. The rest should be the same as for Remote Desktop.
